hugegraph-mysql/src/main/java/com/baidu/hugegraph/backend/store/mysql/MysqlOptions.java

```diff
@@ -97,4 +97,12 @@ public static synchronized MysqlOptions instance() {
                     disallowEmpty(),
                     "disable"
             );
+
+    public static final ConfigOption<String> STORAGE_ENGINE =
+            new ConfigOption<>(
+                    "mysql.engine",
+                    "the storage engine to store graph data/schema, innodb or rocksdb.",
```

Comment:
"The storage engine of backend store database, like InnoDB/MyISAM/RocksDB for MySQL"